Makefile

```diff
@@ -26,6 +26,24 @@ SOURCEC = $(SRC)/main.cpp \
 	$(SRC)/sp.cpp
 OBJECTS = $(SOURCEC:%.cpp=%.o)
 
+MPI_PATH = ${PETSC_DIR}/arch-label-opt/bin
+
+help:
+	@echo ""
+	@echo "Commands:"
+	@echo ""
+	@echo "  all		Build and install Mandyoc by running"
+	@echo "  test_madyoc	Run the Mandyoc test using 2 cores. It takes several munutes"
```

Comment:
Tests now require numpy, so we need to state that on requirements? Maybe add a section for development requirements?